apps/notifications/babel.config.js

```diff
@@ -0,0 +1,54 @@
+// @babel/preset-env modules option. `false` for ECMAScript modules.
```

Comment:
This file is mostly copied from `calypso-babel-config`, with just a few changes.

Reply:
We can delete half of this file as it is not meant to be configurable anymore, just the config for a specific project. Anything related to `opts`, configure modules, etc.

Or have a separate preset in `@automattic/calypso-babel-config` for Preact projects.